pkg/model/resources/nodeup.go

```diff
@@ -131,7 +130,10 @@ function download-release() {
   esac
 
   cd ${INSTALL_DIR}/bin
-  try-download-release
+  local -r nodeup_urls=( $(split-commas "${NODEUP_URL}") )
+  download-or-bust nodeup "${NODEUP_HASH}" "${nodeup_urls[@]}"
+
+  chmod +x nodeup
```

Comment:
Probably best to move url (not used anywhere else) and chmod to `download-or-bust()` and keep `download-release()` clean.
```suggestion
  download-or-bust nodeup "${NODEUP_HASH}" "${nodeup_urls[@]}"
```

Reply:
`download-or-bust` has two return points, so it is simpler to have the chmod outside it. Also, it's not completely impossible that we might later need to download a second, non-executable file before running nodeup.